Makefile

```diff
@@ -27,7 +27,7 @@ BINDIR        ?= bin
 BUILD_DIR     ?= build
 COVERAGE      ?= $(CURDIR)/coverage.html
 SC_PKG         = github.com/kubernetes-incubator/service-catalog
-TOP_SRC_DIRS   = cmd contrib pkg
+TOP_SRC_DIRS   = cmd contrib pkg plugin
```

Comment:
when we have a clear story on generic machinery including admission controllers by default, yes, but right now, no.  i am working with sig-api-machinery folks to get a story here.